http/org.wso2.carbon.transport.http.netty/src/main/java/org/wso2/carbon/transport/http/netty/common/Util.java

```diff
@@ -494,4 +497,25 @@ public static String getSystemVariableValue(String variableName, String defaultV
     public static String createServerConnectorID(String host, int port) {
         return host + ":" + port;
     }
+
+    public static HTTPCarbonMessage createErrorMessage(String payload) {
```

Comment:
Shall we include javaDocs explaining this ?

Reply:
done